com.dynamo.cr/com.dynamo.cr.bob.test/src/com/dynamo/bob/pipeline/LuaScannerTest.java

```diff
@@ -486,4 +486,13 @@ public void testRemoveEmptyLifecycleFunctions() throws Exception {
         assertEquals(expected, parsed);
     }
 
+    @Test
+    public void testSemicolon() throws Exception {
+        // Basic
+        String luaCode = "do return nil end;else end;";
```

Comment:
Didn't you mention the issue also had to do with a semicolon after a go.property() call? Do we need a test specifically for that case?

Reply:
We have such test already in another test file